backends/libvirt/cfg/tests-shared.cfg

```diff
@@ -15,6 +15,7 @@ include subtests.cfg
 include machines.cfg
 include guest-os.cfg
 include guest-hw.cfg
+include host-os.cfg
```

Comment:
The order seems to be okay, but I like to know why not put this in front of `machines.cfg`?